src/main/java/org/example/homework6/MyException.java

```diff
@@ -0,0 +1,7 @@
+package org.example.homework6;
+
+public class MyException extends Exception {
```

Comment:
Nice idea with using own exception, but consider renaming it, e.g 'InteractiveCalculatorException'